BookingsApi/BookingsApi.Common/Logging/AllocationsLogger.cs

```diff
@@ -0,0 +1,15 @@
+namespace BookingsApi.Common.Logging
+{
+    using System;
+    using Microsoft.Extensions.Logging;
+
+    public static partial class AllocationsLogger
```

Comment:
also, just for consistency (sorry to be picky) please could you make them singular names:

AllocationLogger
HearingLogger

Reply:
done